pls_core/src/inflections/snapshots/pls_core__inflections__tests__inflection_tests__conjugation_1_en.snap

```diff
@@ -6,11 +6,8 @@ expression: html
 <div class="pls-inflection-root">
   <header class="pls-inflection-header">
     <summary>
-      <div class="pls-inflection-summary-word-info">^ābādheti$ &ndash; "eti pr" (like ^bhāveti$)</div>
-      <div class="pls-inflection-summary-definition">
-        <span class="pls-inflection-summary-definition-pos">(pr)</span> <span class="pls-inflection-summary-definition-meaning">oppresses, harasses</span>
-      </div>
```

Comment:
this needs to be pulled out of this commit - we're going to fix this differently.

ie host is going to pass an option whether it wants it or not. that way web-ui will have it and dictionary will not
see https://github.com/digitalpalitools/web-ui/issues/27